### ISSUE TITLE
[Feature Request] Word Wrapping

### ISSUE THREAD

**Issue by okabak123** [OliBlade/3Cols#27]
It would be great if you allowed word wrapping!

**Comment by OliBlade** [comment]
Hi okabak, do you mean word wrapping within the editor?

**Comment by okabak123** [comment]
Yes. It would be great as a checkbox option or something toggleable.

**Comment by OliBlade** [comment]
We just added the Word Wrap feature as a setting available from the settings button (see https://docs.3cols.com/theeditor ).

You may need to hard refresh (ctrl + f5) your browser if the setting is not appearing.
